build.gradle

```
@@ -77,6 +77,7 @@ dependencies {
 
     // test
     testImplementation 'org.junit.jupiter:junit-jupiter'
+    testRuntimeOnly("org.junit.platform:junit-platform-launcher")
```

Should we stay consistent with the other lines?
```suggestion
    testRuntimeOnly 'org.junit.platform:junit-platform-launcher'
```

---

I prefer to stick to gradle release notes
https://docs.gradle.org/current/userguide/upgrading_version_8.html#manually_declaring_dependencies